bigtable/admin.go

```diff
@@ -275,6 +278,35 @@ func (ac *AdminClient) CreateColumnFamily(ctx context.Context, table, family str
 	return err
 }
 
+// UpdateTable updates a table in the instance from the given configuration.
+// Only deletion protection can be updated at this moment
+func (ac *AdminClient) UpdateTable(ctx context.Context, conf *TableConf) (updated bool, err error) {
```

Comment:
Why are we using `TableConf` here? Are all the fields in `TableConf` relevant to UpdateTable?

Maybe use a new struct: UpdateTableConf
{
        TableID   string
	// Set to true to make the table protected against data loss
	// i.e. deleting the table, the column families in the table and the instance containing the table would be prohibited
	DeletionProtection *bool
}